src/oncall/scheduler/default.py

```diff
@@ -14,40 +14,40 @@
 SECONDS_IN_A_WEEK = SECONDS_IN_A_DAY * 7
 
 columns = {
-    'id': '`event`.`id` as `id`',
```

Comment:
Why is this needed? Preview shouldn't need these constraints. Also, move this to `preview.py`, so all falcon logic is handled in the API responders